tools/compile_only_aspect.bzl

```diff
@@ -15,15 +15,9 @@ def _compile_only_aspect_impl(target, ctx):
     # Consider in Swift compilation how to only generate the intefaces
     # through the feature of interface splitting
     if ctx.rule.kind == "swift_library":
-        for action in target.actions:
-            if action.mnemonic == "SwiftCompile":
```

Comment:
@congt If the action is a `swift_library` is it possible there are other action types in there or no? This seems fine none the less - my thought was it might add extra actions other than a `SwiftCompile` we dont want

Reply:
I only see one SwiftCompile action. I can revert this line.

Reply:
Reverted